samples/build-calcupad-pipeline.yaml

```diff
@@ -32,8 +52,8 @@ spec:
           workspace: shared-data
       params:
         - name: url
-          value: https://github.com/audiokit/AudioKit.git
-    - name: build-audiokit
+          value: https://github.com/kwonye/calcupad.git
```

Comment:
We should checkout a specific branch or tag for the repo. We should be able to do this with the `revision` param - https://github.com/tektoncd/catalog/blob/main/task/git-clone/0.4/git-clone.yaml#L46

Reply:
pinned a specific commit hash since all tags still weren't working (older compatibility issues).  And there's not really any release branches